tools/maf/vcf_to_maf_customtrack.py

```diff
@@ -13,51 +13,43 @@
 
 class PopulationVCFParser(Iterator):
     def __init__(self, reader, name):
-        self.reader = reader
+        self.reader = iter(reader)
```

Comment:
A tangent. I'm wondering... We have to do this because the reader's `__iter__` method does not return `self` (i.e., it's not an iterator without that), right? Would it makes sense to refactor that code so that it doesn't require this step (wrapping in `iter()` and, thus, is more straightforward to use?

Reply:
Sure, why not, but I don't think that would be a good use of time. These tools are legacy and I doubt anyone would really build new stuff based on galaxy-sequence-utils.

Reply:
Oh OK, makes sense. Never mind then, there's only so much time.